frontend/src/libraries/explorer-wamp/transactions.ts

```diff
@@ -105,6 +105,33 @@ export interface ReceiptOutcome {
 export interface ReceiptsOutcomeWrapper {
   receiptsOutcome?: ReceiptOutcome[];
 }
+export interface ReceiptsList {
```

Comment:
This type is not a list, so should not be called with "List" suffix. It seems that it is an internal interface just to provide a type to the response we get from the backend, so I suggest we don't `export` it, and call something like `RpcReceipt`.